low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -166,22 +167,39 @@ def _raid_health_monitor(self):
                 # Check RAID device array state is 'idle' or not
                 result = self._check_raid_state(device)
                 if result == "failed":
+                    # Send alert
+                    self.alert_type = self.FAULT
```

Comment:
where are we checking if we had send the fault already for this same device error ? So every iteration we will keep sending same fault, if situation is same.

Reply:
It will store the FAULT state once in a file only.
Then the file will be updated only when state changes to FAULT_RESOLVED